src/styles/var.scss

```diff
@@ -0,0 +1,5 @@
+$card-main-color: #060B35;
+$card-secondary-color: #616070;
+$card-blue-color: #00ACDC;
```

Comment:
- Variable names should describe their purpose. If the designer changes the color to red instead, it will be strange when `$color-blue` will be red. For example `$main-color`